definitions/procedures/pulp/remove.rb

```diff
@@ -0,0 +1,99 @@
+module Procedures::Pulp
+  class Remove < ForemanMaintain::Procedure
+    metadata do
+      description 'Remove pulp2'
+      for_feature :pulp2
+
+      confine do
+        check_min_version('katello', '4.0')
```

Comment:
i would use katello-common here so it can be run on a smart proxy

Reply:
to clarify, katello-common gets installed on external smart proxies and on the katello server itself